back-end/API/app/controllers/user_controller.py

```
@@ -1,257 +1,52 @@
+    dto.id = id
+    updated = user_service.update_name_and_email(current_user, dto)
+    return updated.model_dump(), 200
 
 @user_blueprint.route('/all', methods=['GET'])
 @token_admin_required
-def getAllUsers(current_user):
-    response = user_service.getAllUsers()
-    users = response.json['users']
-    return jsonify({'users': users})
-
+def get_all(current_user):
```

Ruff (0.15.13)

[warning] Unused function argument: `current_user`

(ARG001)